package.json

```diff
@@ -10,21 +10,21 @@
+    "test:acceptance:debug": "mkdir -p build/reports && cross-env SESSION_DETAILS_PORT=8081 OS_PLACES_URI=http://localhost:8090 OS_PLACES_URI=http://localhost:8090 TESTS=acceptance node --inspect-brk=9229 $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/acceptance/features/ --tags 'not @ignore' --require 'src/test/common/steps/**/*.js' --require 'src/test/acceptance/config' --require 'src/test/common/timeout.js' -f json:build/reports/acceptance-report.json",
     "test:acceptance:report": "node src/test/acceptance/acceptance-report.js",
     "test:smoke": "npm-run-all -s test:smoke:test test:smoke:report",
     "test:smoke:test": "mkdir -p build/reports && cross-env TESTS=smoke cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/**/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
     "test:smoke:debug": "mkdir -p build/reports && cross-env TESTS=smoke node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/**/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
     "test:smoke:report": "node src/test/smoke/smoke-report.js",
     "test:compatibility:local": "npm-run-all -s test:compatibility:local:test test:compatibility:local:report",
-    "test:compatibility:local:test": "mkdir -p build/reports/compatibility && cross-env SESSION_DETAILS_PORT=8081 OS_PLACES_URI=http://localhost:8090 TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/**/*.js' --require 'src/test/common/timeout.js' --require 'src/test/compatibility/config/environment.js' -f json:build/reports/compatibility-report.json",
-    "test:compatibility:local:debug": "mkdir -p build/reports/compatibility && cross-env SESSION_DETAILS_PORT=8081 OS_PLACES_URI=http://localhost:8090 TESTS=localCompatibility node --inspect-brk=9229 $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/**/*.js' --require 'src/test/common/timeout.js' --require 'src/test/compatibility/config/environment.js' -f json:build/reports/compatibility-report.json",
+    "test:compatibility:local:test": "mkdir -p build/reports/compatibility && cross-env SESSION_DETAILS_PORT=8081 OS_PLACES_URI=http://localhost:8090 OS_PLACES_URI=http://localhost:8090 TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/**/*.js' --require 'src/test/common/timeout.js' --require 'src/test/compatibility/config/environment.js' -f json:build/reports/compatibility-report.json",
```

Comment:
Duplicate OS_PLACES_URI in test:compatibility:local:test, test:acceptance:debug and test:compatibility:local:debug

Reply:
Removed duplicates.